### ISSUE TITLE
Incompatibility between chemformula and russian language

### ISSUE THREAD

**Issue by deOliveira-R** [reutenauer/polyglossia#465]
I happen to have run into an error while writting my thesis and it seems like the russian language somehow defines a command \ch, which conflicts with the chemformula package. The result is an error saying that command \ch is already defined.

I had the same problem with babel. At some point I had the code "\let\ch\relax" after importing the language package, but it does not work anymore since I updated the system. Is it really necessary for it to be \ch? If I'm not mistaken, this is for cosinus hyperbolicus, so couldn't it be \cosh or something like that as we usually write in math? Or maybe a smarter way?

**Comment by jspitz** [comment]
Workaround:
```tex
% !TeX TS-program = xelatex
\documentclass{article}

\usepackage{chemformula}
% Store chemformula's definition
\let\origch\ch

\usepackage{polyglossia}
\setmainlanguage{russian}

% restore chemformula's definition
\AtBeginDocument{\let\ch\origch}

\setmainfont{DejaVu Serif}
\begin{document}
	
\ch{H2O}
	
\end{document}
```

As to your request: I don't think we should rename these functions (for backwards compatibility reasons). However, I think we should add an option to disable the definition of these math function, e.g. `mathfunctions=false`. This possibility is also mentioned in the code:

https://github.com/reutenauer/polyglossia/blob/4b8fd9e88fd1d4d6311ccde422b7078e1fb61007/tex/gloss-russian.ldf#L356-L357

**Comment by jspitz** [comment]
Option implemented for next version (1.52).
